src/services/hooks.rs

```diff
@@ -0,0 +1,7 @@
+use crate::utils::context::Context;
+use anyhow::Result;
+
+pub fn invalidate_caches(context: &Context, safe: &String) -> Result<()> {
+    println!("{}", safe);
```

Comment:
Same as above, remove or change to `dbg!`?

Reply:
Same as above